docs/src/user-guide/configuring/configuration-files-new.md

```diff
@@ -547,7 +549,7 @@ Here, the second configuration object only overrides the severity, so the final
 
 ### Configuring shared settings
 
-ESLint supports adding shared settings into configuration files. Plugins use `settings` to specify information that should be shared across all of its rules. You can add a `settings` object to a configuration object and it will be supplied to every rule being executed. This may be useful if you are adding custom rules and want them to have access to the same information. Here's an example:
+ESLint supports adding shared settings into configuration files. Plugins use `settings` to specify information that should be shared across all of its rules. When you add a `settings` object to a configuration object, and it is supplied to every rule executed by that configuration object. This may be useful if you are adding custom rules and want them to have access to the same information. Here's an example:
```

Comment:
```suggestion
ESLint supports adding shared settings into configuration files. Plugins use `settings` to specify the information that should be shared across all of its rules. When you add a `settings` object to a configuration object, and it is supplied to every rule executed by that configuration object. This may be useful if you are adding custom rules and want them to have access to the same information. Here's an example:
```

Reply:
this change looks good.

but upon re-reading the sentence, i think there's something wrong with the 'across all **its** rules' phrase. what does 'its' refer to?

@snitin315 or @nzakas, do you know?

Reply:
I believe `it` is referring to the `Plugin` here.

Reply:
That’s correct. Note: settings are not limited to one plugin. Every plugin receives every setting that was set in the config. It’s a convention that plugins namespace the settings they are interested in to avoid collisions with others.

Reply:
thanks for the clarifications. i believe i've addressed this confusion in the latest commit, [53dd21d](https://github.com/eslint/eslint/pull/16509/commits/53dd21d21609449d6fa5a965fd967fa48606b1ef).

also added the word 'the' before information that started this conversation.